pom.xml

```diff
@@ -18,51 +18,51 @@
   <modules>
     <module>bom</module>
     <module>parent</module>
-    <module>atomix</module>
-    <module>auth</module>
-    <module>broker</module>
-    <module>broker-client</module>
-    <module>protocol-test-util</module>
     <module>dist</module>
```

Comment:
❓ Does the dist folder contain anything non-zeebe? Or is the intention that other components will use it as well?

Reply:
Soon it will contain also other components. This will be the place where we will create the single jar.